GDAS-validation/gdas_config/config.anal

```diff
@@ -0,0 +1,155 @@
+  fi
+
+  #   Place GOES-15 AMVs in monitor, assimilate GOES-17 AMVs, assimilate KOMPSAT-5 gps
+  if [[ "${PDY}${cyc}" -ge "2019110706" && "${PDY}${cyc}" -lt "2020040718" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2019110706
+    export OBERROR=${FIXgsi}/gfsv16_historical/prepobs_errtable.global.2019110706
+  fi
+
+  #   Assimilate 135 (T) & 235 (uv) Canadian AMDAR observations
+  if [[ "${PDY}${cyc}" -ge "2020040718" && "${PDY}${cyc}" -lt "2020052612" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2020040718
+    export OBERROR=${FIXgsi}/gfsv16_historical/prepobs_errtable.global.2020040718
+  fi
+
+  #   Assimilate COSMIC-2
+  if [[ "${PDY}${cyc}" -ge "2020052612" && "${PDY}${cyc}" -lt "2020082412" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2020052612
+    export OBERROR=${FIXgsi}/gfsv16_historical/prepobs_errtable.global.2020040718
+  fi
+
+  #   Assimilate HDOB
+  if [[ "${PDY}${cyc}" -ge "2020082412" && "${PDY}${cyc}" -lt "2020091612" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2020082412
+  fi
+
+  #   Assimilate Metop-C GNSSRO
+  if [[ "${PDY}${cyc}" -ge "2020091612" && "${PDY}${cyc}" -lt "2021031712" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2020091612
+  fi
+
+  #   Assimilate DO-2 GeoOptics
+  if [[ "${PDY}${cyc}" -ge "2021031712" && "${PDY}${cyc}" -lt "2021091612" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2021031712
+  fi
+
+  #   NOTE:
+  #   As of 2021110312, gfsv16_historical/global_convinfo.txt.2021110312 is
+  #   identical to ../global_convinfo.txt.  Thus, the logic below is not
+  #   needed at this time.
+  #   Assimilate COSMIC-2 GPS
+  #   if [[ "${PDY}${cyc}" -ge "2021110312" && "${PDY}${cyc}" -lt "YYYYMMDDHH" ]]; then
+  #   export CONVINFO=$FIXgsi/gfsv16_historical/global_convinfo.txt.2021110312
+  #   fi
+
+  #   Turn off assmilation of OMPS during period of bad data
+  if [[ "${PDY}${cyc}" -ge "2020011600" && "${PDY}${cyc}" -lt "2020011806" ]]; then
+    export OZINFO=${FIXgsi}/gfsv16_historical/global_ozinfo.txt.2020011600
+  fi
+
+
+  #   Set satinfo for start of GFS v16 parallels
+  if [[ "${PDY}${cyc}" -ge "2019021900" && "${PDY}${cyc}" -lt "2019110706" ]]; then
+    export SATINFO=${FIXgsi}/gfsv16_historical/global_satinfo.txt.2019021900
+  fi
+
+  #   Turn on assimilation of Metop-C AMSUA and MHS
+  if [[ "${PDY}${cyc}" -ge "2019110706" && "${PDY}${cyc}" -lt "2020022012" ]]; then
+    export SATINFO=${FIXgsi}/gfsv16_historical/global_satinfo.txt.2019110706
+  fi
+
+  #   Turn off assimilation of Metop-A MHS
+  if [[ "${PDY}${cyc}" -ge "2020022012" && "${PDY}${cyc}" -lt "2021052118" ]]; then
+    export SATINFO=${FIXgsi}/gfsv16_historical/global_satinfo.txt.2020022012
+  fi
+
+  #   Turn off assimilation of S-NPP CrIS
+  if [[ "${PDY}${cyc}" -ge "2021052118" && "${PDY}${cyc}" -lt "2021092206" ]]; then
```

Comment:
Oops, same thing happens here for satinfo.  This logic replaces the fix satinfo with a historical satinfo when running 2021080100 gdasanal.  We wind up with
```
+ exglobal_atmos_analysis.sh[295]: SATINFO=/work2/noaa/da/rtreadon/gdas-validation/global-workflow/fix/gsi/gfsv16_historical/global_satinfo.txt.2021052118
```
in gdasanal.log.

Reply:
Addressed via 37fea66